sensor/docs/tutorials/tag_based_pnp_calibrator.md

```diff
@@ -0,0 +1,153 @@
+
+## Launching the tool
+
+In this tutorial, we will use the X2 project as an example.
+First, run the sensor calibration manager:
+
+```text
+ros2 run sensor_calibration_manager sensor_calibration_manager
+```
+
+In `project`, select `x2`, and in `calibrator`, select `tag_based_pnp_calibrator`. Then, press `Continue`.
+
+<p align="center">
+    <img src="../images/tag_based_pnp_calibrator/menu1.jpg"  alt="menu1">
+</p>
+
+A menu titled `Launcher configuration` should appear in the UI, and the user may change any parameter he deems convenient.
+For this tutorial, we will modify the default values `calibration_pairs` from `9` to `8` as the bag has 8 AprilTag detections and also modify the `camera_name` from `camera0` to `camera6`. After configuring the parameters, click `Launch`.
+
+<p align="center">
+    <img src="../images/tag_based_pnp_calibrator/menu2.jpg"  alt="menu2">
+</p>
+
+The following UI should be displayed. When the `Calibrate` button becomes available, click it.
+If it does not become available, it means that either the required `tf` or services are not available.
+
+In this tutorial, since the `tf` are published by the provided rosbag, start by running the bag with the command `ros2 bag play camera_lidar.db3 --clock -r 0.1`. Afterward, launch the tools and click the `Calibrate` button."
+
+<p align="center">
+    <img src="../images/tag_based_pnp_calibrator/menu3.jpg"  alt="menu3">
+</p>
+
+## Calibration
+
+The calibration starts automatically after clicking the `Calibrate` button. It will keep calibrating the LidarTag detections and AprilTag detections until the number of the detections fits the user-defined `calibration_pairs` in the `Launcher configuration`.
+
+When the user starts the calibration, `rviz` and the `image view` should be displayed like below.
+
+<p align="center">
+    <img src="../images/tag_based_pnp_calibrator/visualization1.jpg"  alt="visualization1">
+</p>
+
+After the tools detect the LidarTag and AprilTag, it will show the detection markers on the `rviz` and the `image view`. The text in the rviz will also display the current number of pairs of LidarTag detections and AprilTag detections.
+
+<p align="center">
+    <img src="../images/tag_based_pnp_calibrator/visualization2.jpg"  alt="visualization2">
+</p>
+
+Once the user observes the number of pairs increases in the `rviz`, he can start moving the tag to another position.
+
+While moving the tag to another position, the user should move the tag decisively from one place to the other, not stopping and starting mid-way.
+Please make sure the moving distance is larger than the `calibration_min_pair_distance` and the tag is in the FOV of both lidar and camera.
```

Comment:
This one is not clear.
What you what to make the user understand is that the new position should be at least `calibration_min_pair_distance` from the closest detection in the set of converged detections

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/ea47fc19844e40b5e24e146866ed44b4d8bfae98

Reply:
We are getting close to the realm of nits, but there is a grammatical error here:

`Please make sure the new position should be at least`
to
`Please make sure the new position is at least`

Reply:
@vividf

Reply:
thanks, fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/2338794d32e030eded8b97a2f9ee3d7e7917a1a8